seed/views/properties.py

```diff
@@ -711,6 +711,258 @@ def update(self, request, pk=None):
+              paramType: form
+            - name: remove_measures
+              description: list of measure_ids or measure long names to remove from property
+              type: array
+              required: false
+              paramType: form
+            - name: implementation_status
+              description: Enum on type of measures. Recommended, Proposed, Implemented
+              required: true
+              paramType: form
+              type: string
+              enum: ["Recommended", "Proposed", "Implemented"]
+        """
+        cycle_pk = request.query_params.get('cycle_id', None)
+        if not cycle_pk:
+            return JsonResponse(
+                {'status': 'error', 'message': 'Must pass cycle_id as query parameter'})
+
+        implementation_status = PropertyMeasure.str_to_impl_status(
+            request.data.get('implementation_status', None))
+        if not implementation_status:
+            return JsonResponse(
+                {'status': 'error', 'message': 'None or invalid implementation_status type'}
+            )
+
+        result = self._get_property_view(pk, cycle_pk)
+        pv = None
+        if result.get('status', None) != 'error':
+            pv = result.pop('property_view')
+        else:
+            return JsonResponse(result)
+
+        # get the list of measures to add/remove and return the ids
+        add_measure_ids = Measure.validate_measures(request.data.get('add_measures', []).split(','))
+        remove_measure_ids = Measure.validate_measures(
+            request.data.get('remove_measures', []).split(','))
+
+        # add_measures = request.data
+        message_add = []
+        message_remove = []
+        message_existed = []
+
+        property_state_id = pv.state.pk
+
+        for m in add_measure_ids:
+            join, created = PropertyMeasure.objects.get_or_create(
+                property_state_id=property_state_id,
+                measure_id=m,
+                implementation_status=implementation_status
+            )
+            if created:
+                message_add.append(m)
+            else:
+                message_existed.append(m)
+
+        for m in remove_measure_ids:
+            qs = PropertyMeasure.objects.filter(property_state_id=property_state_id,
+                                                measure_id=m,
+                                                implementation_status=implementation_status)
+            if qs.exists():
+                qs.delete()
+                message_remove.append(m)
+
+        return JsonResponse(
+            {
+                "status": "success",
+                "message": "Updated measures for property state",
+                "added_measure_ids": message_add,
+                "removed_measure_ids": message_remove,
+                "existing_measure_ids": message_existed,
+            }
+        )
+
+    # TODO: fix the url_path to be nested. I want the url_path to be measures and have get,post,put
```

Comment:
yeah, I spun my wheels on this way too long. If I were to just switch the verb in the function, then swagger isn't happy because our version of swagger (0.3.1) doesn't support documenting various verbs in a method.